plugins/header_rewrite/header_rewrite.cc

```diff
@@ -304,8 +325,8 @@ RulesConfig::parse_config(const std::string &fname, TSHttpHookID default_hook, c
   }
 
   // Add the last rule (possibly the only rule)
-  if (add_rule(rule.get())) {
-    rule.release();
+  if (rule && rule->has_operator()) {
+    add_rule(std::move(rule)); // Transfer ownership
```

Comment:
No need for `// Transfer ownership` comment, that's clear from `std::move` on a `unique_ptr`.

Reply:
I actually still see the commet in the github display of the patch.